### PR TITLE
YSHsieh7777 term proposal

### PR DESCRIPTION
**Pull request by YSHsieh7777**

### CODE REVIEW

File: term_proposal/YSHsieh7777/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ - **Simulate and visualize** the situation that crowd evacuate from indoor to outdoor.
+ - Take **human behaviors** into consideration to determine crowd's action.
+ - Let user **design** the indoor traffic flow.
+ - Calculate the amount of time that people evacuate from indoor to outdoor.
+ - Calculate the number of people that die in the fire. 
+ - Store the **best** traffic flow.
+
+## To-Do List
+
+ - [x] Find Term Project topic. ( 1 week )
```
Comment (yungyuc) [inline term_proposal/YSHsieh7777/README.md:33]: You are well organized.

File: term_proposal/YSHsieh7777/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ - [x] Collect information on that topic. ( 1-2 weeks )
+ - [x] Determine the targets of the project. ( 1 week )
+ - [ ] Plan and build the project structure. ( 1-2 week )
+ - [ ] Coding and debugging. ( 4-6 weeks )
+ - [ ] Refine features and codes. ( 1-2 weeks ) 
+ - [ ] Prepare the presentation. ( 1 week )
+
+## Example
```
Comment (yungyuc) [inline term_proposal/YSHsieh7777/README.md:41]: You should describe how you are architecting the system.  Nothing may be crystal clear at this point, but attempting to specify will help you down the road.

Include things like language, data structure, system facilities, when they apply.

**Comment by yungyuc** [comment]
I am taking the proposal so the timestamp looks good for you.  But it's fine if you want to make a second PR to address my comments.  Don't worry about being behind the deadline.  The primary purpose of the proposal is to facilitate the execution of your project.

**Comment by YSHsieh7777** [comment]
Thank you for your suggestion! I will take it to refine my proposal and make a second PR. Thanks!

**Pull request merged**